packages/typedoc-plugins/tests/module-fixer/fixtures/logger.ts

```
@@ -0,0 +1,16 @@
+
+import type Error from './error';
+
+/**
+ * @module fixtures/logger
+ */
+
+export default class Logger {
+	public static error( error: Error ): void {
+		console.log( error.name );
+	}
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)